integration/api/src/main/java/com/cerner/beadledom/integration/api/HelloWorldResource.java

```diff
@@ -0,0 +1,35 @@
+package com.cerner.beadledom.integration.api;
+
+import com.cerner.beadledom.integration.api.model.HelloWorldDto;
+import com.cerner.beadledom.integration.api.model.OffsetPaginatedListHelloWorldDto;
+import com.cerner.beadledom.jaxrs.GenericResponse;
+import io.swagger.annotations.Api;
```

Comment:
For the client resource I think the swagger annotations can all be removed.

Reply:
I removed them when I moved the resource to the client: https://github.com/cerner/beadledom/pull/149/commits/acc0165095c733471eec959e3a986d962a3146e2